src/libecalc/core/models/generator.py

```diff
@@ -10,10 +10,16 @@ def __init__(
         self,
         data_transfer_object: dto.GeneratorSetSampled,
     ):
+        fuel_values = data_transfer_object.fuel_values
```

Comment:
We are probably doing it the same way other places, but I think that this "manipulation" of the data should be done earlier, when reading the data. Why? Because we alter the data(-file), and not the model itself. If the data file is used other places/for other things, it should still be applied. In theory a file can be used more than one place. Please let me know what you think.

Reply:
Although, the way it has been implemented, sending the adjustment factor and constant to the genset, it makes sense to put here...